site/docs/v1/tech/apis/_apikey-update-response-body.adoc

```diff
@@ -0,0 +1,32 @@
+==== Response Body
+
+[.api]
+[field]#apiKey.metaData.attributes.description# [type]#[String]#::
+Description of the key.
+
+[field]#apiKey.insertInstant# [type]#[Long]#::
+The link:/docs/v1/tech/reference/data-types/#instants[instant] when the API key was added to the FusionAuth database.
+
+[field]#apiKey.key# [type]#[String]#::
+The API key value, this is the string value you will use in the Authorization header to link:/docs/v1/tech/apis/authentication/[authenticate API] requests.
+
+[field]#apiKey.keyManager# [type]#[Boolean]#::
+This the keymanager is set to false, this API key can be used to call the API key API. Otherwise, this API key will always return a 401 when you try to use this API.
```

Comment:
Typo? "This the keymanager"?

Also, if you are referring to fields, please use `[field]#keyManager#`, as that does some nice formatting.